xxh3.h

```diff
@@ -578,7 +588,7 @@ XXH3_len_9to16_64b(const xxh_u8* input, size_t len, const xxh_u8* secret, XXH64_
 {
```

Comment:
Shouldn't you rename it to `XXH3_len_8to16_64b`?

Reply:
yeah, either this, or revert `len==8` into `4to8`.
nothing decided yet, this is just test.

Reply:
Ok